generators/PrintData.lua

```diff
@@ -143,7 +143,7 @@ function parseArgs()
     local overwrite = true
     for i = 1, #arg do
         a = arg[i]
-        m, n = a:match('(%a+)=(%a+)')
+        m, n = a:match('([^=]+)=([^=]+)')
```

Comment:
I think this should be something like:

'([^=]+)=(.+)'

There's no reason to disallow the equal sign in the value.

Reply:
indeed